tools/StandardAnchorTags/Program.cs

```diff
@@ -141,7 +141,7 @@ private static async Task<string> ReadExistingReadMe()
             // This is the first node in the TOC, so truncate here:
             var index = contents.IndexOf(TOCHeader);
 
-            return contents.Substring(0, index);
+            return (index == -1) ? contents : contents.Substring(0, index);
```

Comment:
This strictly isn't necessary, but it means that we can remove the TOC nested list by hand, and the next run will add it. It made these changes easier to test, and seems like a nice to have.

Reply:
Again, I think this can go away entirely :)